laws/src/main/scala/cats/laws/UnorderedFoldableLaws.scala

```diff
@@ -69,6 +69,8 @@ trait UnorderedFoldableLaws[F[_]] {
   def nonEmptyRef[A](fa: F[A]): IsEq[Boolean] =
     F.nonEmpty(fa) <-> !F.isEmpty(fa)
 
+  def containsConsistentWithExists[A](fa: F[A], v: A)(implicit eq: Eq[A]): IsEq[Boolean] =
+    F.contains(fa)(v) <-> F.forall(fa)(a => eq.eqv(a, v))
```

Comment:
Nice!

Reply:
Oh, should that `forall` be `exists`? 🤔

Reply:
🙈

Reply:
Yes, I think the tests were passing because in none of the cases `v` belongs to `fa`. Then, both sides of the equivalence are always false. It may be possible to fix it with some creative manipulation of the arbitraries in the discipline tests.

Reply:
I'd suggest to add more laws then:
```scala
def containsConsistentWithForall[A](fa: F[A], v: A)(implicit eq: Eq[A]): IsEq[Boolean] =
  !F.contains(fa)(v) <-> F.forall(fa)(a => eq.neqv(a, v))
```

Reply:
Also I think it might be useful to add a synthetic law like this one:
```scala
  def YOU_NAME_IT[A](fa: F[A]): Boolean = {
    F.unorderedFoldMap(fa)(F.contains(fa)(_))(new CommutativeMonoid[Boolean] {
      override def combine(x: Boolean, y: Boolean): Boolean = x && y
      override def empty: Boolean = true
    })
  }
```
I.e. it should check that `contains_` returns `true` for each `A` from `F[A]`.

Perhaps, that law is a controversial one, but let's discuss it :)

Reply:
@satorg I like that law, but I think it was easier to implement using `forall` :)

Reply:
Sure it was :)